CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Setup
 
-To start developing on AWM Relayer, you'll need Golang v1.22.6.
+To start developing on AWM Relayer, you'll need Golang v7.
```

Comment:
Would it be more clear to stick with the 1.22.7 naming for Golang versions? The golang release page doesn't seem to refer to 1.22.7 as v7 (https://go.dev/dl/)

Reply:
Oops, definitely a typo